beacon-chain.k

```diff
@@ -1398,12 +1406,16 @@ def get_attestation_deltas(state: BeaconState) -> Tuple[Sequence[Gwei], Sequence
                        UnslashedIndices,
                        AttestingBalance,
                        #mapMapPair(Rewards => #if contains(I, UnslashedIndices)
-                                              #then Rewards[I <- ({Rewards[I]}:>Int +Int get_base_reward(I)
-                                                                  *Int AttestingBalance /Int TotalBalance)]
+                                              #then Rewards[I <- ei(0, 2 ^Int 64 -Int 1, ci({Rewards[I]}:>Int) +UInt64 get_base_reward(I)
+                                                                  *UInt64 ( ui(0, TOTAL_ETHER_SUPPLY, AttestingBalance)
+                                                                  /UInt64 ci(EFFECTIVE_BALANCE_INCREMENT))
+                                                                  /UInt64 ( ui(EFFECTIVE_BALANCE_INCREMENT, TOTAL_ETHER_SUPPLY, TotalBalance)
+                                                                  /UInt64 ci(EFFECTIVE_BALANCE_INCREMENT)))
+                                                                  ]
                                               #else Rewards
                                               #fi,
                                   Penalties => #if notBool contains(I, UnslashedIndices)
-                                               #then Penalties[I <- ({Penalties[I]}:>Int +Int get_base_reward(I))]
+                                               #then Penalties[I <- ei(0, 2 ^Int 64 -Int 1, (ci({Penalties[I]}:>Int) +UInt64 get_base_reward(I)))]
```

Comment:
Same here: `ci({Penalties[I]}:>Int)` should be changed to `ui(MIN, MAX, {Penalties[I]}:>Int)`?